pom.xml

```diff
@@ -43,8 +43,8 @@
   </scm>
   <developers>
     <developer>
-      <id>gabrielasmeria</id>
-      <name>Gabriela Smeria</name>
+      <id>acotiuga</id>
+      <name>Alexandru Cotiuga</name>
```

Comment:
@acotiuga You should not remove past developers unless it was an error. In this case Gabriela did something and it was warranted that she was one of the developers. The fact that the code has been modified doesn't remove the fact that she worked on this extension in the past.